CMakeLists.txt

```
@@ -115,3 +115,5 @@ if(BUILD_TESTING)
 endif()
 
 include(AddUninstallTarget)
+
+export(PACKAGE ${PROJECT_NAME})
```

Following the @traversaro suggestion:
```suggestion
  # Export build directory if CMAKE_EXPORT_PACKAGE_REGISTRY is set.
  # CMake >= 3.15 already checks for CMAKE_EXPORT_PACKAGE_REGISTRY in `export(PACKAGE)` (cf.
  # cf. https://cmake.org/cmake/help/latest/policy/CMP0090.html), and we effectively back-port
  # this behavior to earlier versions.
  if(CMAKE_EXPORT_PACKAGE_REGISTRY OR NOT CMAKE_VERSION VERSION_LESS 3.15)
    export(PACKAGE ${PROJECT_NAME})
  endif()
```

---

Note that that code turned out to be bugged, see https://github.com/robotology/ycm/pull/314 .

---

Thanks, I updated the suggestion accordingly.